### ISSUE TITLE
rcloud frequently hangs on loading new notebook

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud#1307]
this is a new one

**Comment by s-u** [comment]
when changing notebooks?

**Comment by gordonwoodhull** [comment]
yes, i.e. when creating a new session. haven't seen it for a few hours but it was biting me all morning.

**Comment by gordonwoodhull** [comment]
I seem to get the familiar

```
WebSocket is already in CLOSING or CLOSED state.
```

when this happens.  I don't think I have been switching notebooks very fast though.  Maybe I am clicking on a notebook while it's doing something else.
